app.js

```diff
@@ -1,78 +1,78 @@
+// book class - represents a book
 class Book {
   constructor(id, title, author) {
     this.id = id;
     this.title = title;
     this.author = author;
   }
-}
-
-// get books from localstorage
-const getBooks = () => {
-  let books;
 
-  if (localStorage.getItem('books') === null) {
-    books = [];
-  } else {
-    books = JSON.parse(localStorage.getItem('books'));
-  }
+  // get books from localstorage
+  static getBooks = () => {
+    let books;
 
-  return books;
-};
-
-// function to add a new book to the collection, with title and author.
-const addBook = (book) => {
-  const books = getBooks();
-  books.push(book);
-  localStorage.setItem('books', JSON.stringify(books));
-};
-
-// add submit book to browser view
-const addBookToList = (book) => {
-  const list = document.getElementById('bookList');
-
-  const row = document.createElement('div');
-  row.innerHTML = `
-    <div>
-    <div>${book.title}</div>
-    <div>${book.author}</div>
-    <div hidden>${book.id}</div>
-    <button class="delete">Remove</button>
-    <hr>
-    </div>
-  `;
-
-  list.appendChild(row);
-};
-
-// display books
-const displayBooks = () => {
-  const books = getBooks();
-  books.forEach((book) => addBookToList(book));
-};
-
-// remove book from browser view
-const deleteBook = (el) => {
-  if (el.classList.contains('delete'));
-  el.parentElement.remove();
-};
-
-// delete book from local storage
-const removeBook = (id) => {
-  const books = getBooks();
-
-  books.forEach((book, index) => {
-    if (book.id === id) {
-      books.splice(index, 1);
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
     }
-  });
 
-  localStorage.setItem('books', JSON.stringify(books));
-};
+    return books;
+  };
+
+  // display books
+  static displayBooks = () => {
+    const books = Book.getBooks();
+    books.forEach((book) => Book.addBookToList(book));
+  };
+
+  // add submitted book to browser view
+  static addBookToList = (book) => {
+    const list = document.getElementById('bookList');
+
+    const row = document.createElement('div');
+    row.innerHTML = `
+      <div>
+      <div>${book.title}</div>
+      <div>${book.author}</div>
+      <div hidden>${book.id}</div>
+      <button class="delete">Remove</button>
+      <hr>
+      </div>
+    `;
+
+    list.appendChild(row);
+  };
+
+  // delete book from the browser view
+  static deleteBook = (el) => {
+    if (el.classList.contains('delete'));
+    el.parentElement.remove();
+  };
+
+  // function to add a new book to the collection, with title and author.
+  static addBook = (book) => {
+    const books = Book.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  };
+
+  // delete book from local storage
+  static removeBook = (id) => {
+    const books = Book.getBooks();
+
+    books.forEach((book, index) => {
+      if (book.id === id) {
+        books.splice(index, 1);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `slice` over `splice`, which doesn't mutate the array.